lib/mergeIndDisplay.ml

```diff
@@ -95,11 +96,20 @@ let print_differences conf base branches p1 p2 =
       match Image.get_portrait conf base p with
       | Some (`Url url) ->
           ({|<img src="|} ^<^ escape_html url
-           ^>^ {|" style="max-width:75px;max-height:100px">|}
+           ^>^ {|" style="max-width:75px;max-height:100px">|} ^ url
             :> Adef.safe_string)
       | Some (`Path path) ->
-          (* TODO: ?? *)
-          (escape_html path :> Adef.safe_string)
+          let k = Image.default_portrait_filename base p in
+          let s = Unix.stat path in
+          Printf.sprintf
+            {|<img src="%sm=IM&d=%s&%s&k=%s" \
+              style="max-width:75px;max-height:100px"> %s|}
+            (commd conf :> string)
+            (string_of_int
+            @@ int_of_float (mod_float s.Unix.st_mtime (float_of_int max_int)))
```

Comment:
why do we need this information to make src from a path?

Reply:
This is how portraits are displayed. Not sure I remember the reasons for the various parameters